src/pages/Upload/Vcs/index.jsx

```diff
@@ -88,7 +88,7 @@ const UploadFromVcs = () => {
   let uploadId;
 
   // Data required for creating the upload
-  const [uploadVcsData, setUploadFileData] = useState(initialState);
+  const [uploadVcsData, setUploadVcsData] = useState(initialState);
```

Comment:
Can you please explain this change? Its bit confusing to me.

Reply:
Actually I checked that variable name is not appropriate, for it we are doing upload by vcs, so its change nothing technically but just more readable.

Reply:
Ah sorry, I missed the `set`. I thought its `[uploadVcsData, uploadVcsData]` :no_mouth: